### PR TITLE
test: conn_can: add remote request test

### PR DESCRIPTION
**Pull request by Ciusss89**
### Contribution description
Add command to send a can frame with the RTR (remote request) flag on.

### CODE REVIEW

File: tests/conn_can/main.c [hunk 1]
```diff
@@ -114,15 +115,28 @@ static int _send(int argc, char **argv)
         return 1;
     }
 
-    frame.can_id = strtoul(argv[3], NULL, 16);
-    frame.can_dlc = argc - 4;
+    if(not_rtr) {
```
Comment (benpicco) [inline tests/conn_can/main.c:118]:
```suggestion
    if (not_rtr) {
```

File: tests/conn_can/main.c [hunk 2]
```diff
@@ -114,15 +115,28 @@ static int _send(int argc, char **argv)
...
+        frame.can_dlc = strtoul(argv[4], NULL, 10);
+    }
     if (frame.can_dlc > 8) {
         puts("Invalid length");
         return 1;
     }
-    for (int i = 0; i < frame.can_dlc; i++) {
-        frame.data[i] = strtol(argv[4 + i], NULL, 16);
+
+    if(not_rtr) {
```
Comment (benpicco) [inline tests/conn_can/main.c:130]:
```suggestion
    if (not_rtr) {
```

File: tests/conn_can/README.md [hunk 3]
```diff
@@ -78,6 +78,13 @@ To send a raw CAN frame, id 0x100 with 2 bytes of data 01 02 on interface 0:
 test_can send 0 100 01 02
 ```
 
+To send a raw CAN remote request frame, id 0x100 on interface 0 and datalenght 7:
+```
+test_can sendrtr 0 100 7
+```
+A remote request frame has a NULL payload but can have a specific DLC (0 to 8).
```
Comment (benpicco) [inline tests/conn_can/README.md:85]: What is a DLC?

**Comment by Ciusss89** [comment]
> To me it's not immediately clear what `rtr` stands for - please add a comment.

I added a readme, tell me If I have to add details

File: tests/conn_can/README.md (hunk 3 above)
Comment (Ciusss89) [reply tests/conn_can/README.md]: data length code (DLC). I added it to Readme

File: tests/conn_can/main.c [hunk 4]
```diff
@@ -101,7 +102,7 @@ static int _list(int argc, char **argv) {
     return 0;
 }
 
-static int _send(int argc, char **argv)
+static int _send(int argc, char **argv, bool not_rtr)
```
Comment (benpicco) [inline tests/conn_can/main.c:105]: Instead of `not_rtr`, I think it would be simpler to just call it `bool rtr`

Comment (Ciusss89) [reply tests/conn_can/main.c]: should be ok now.

**Pull request merged**